projects/ccd-case-ui-toolkit/src/lib/shared/domain/case-file-view/case-file-view-document.model.ts

```diff
@@ -3,5 +3,5 @@ export class CaseFileViewDocument {
   public document_filename: string;
   public document_binary_url: string;
   public attribute_path: string;
-  public upload_timestamp: string;
+  public upload_timestamp: Date;
```

Comment:
@MateiHMCTS - The upload_timestamp is not implemented yet by CCD. I looked at other objects containing Date returned by CCD, and set the data type accordingly.

This might need changes in the future once upload_timestamp is made available by CCD.

Reply:
@johnbenjamin-hmcts @MateiHMCTS - one suggestion is to declare this as `upload_timestamp: Date | string`. (We have done this in the domain model for Case Flags.)